skaffold.yaml

```diff
@@ -10,4 +10,9 @@ build:
 deploy:
   kubectl:
     manifests:
-      - deploy/service.yaml
+    - deploy/service.yaml
+  # helm:
+  #   releases:
+  #   - name: prometheus-service
+  #     chartPath: charts/prometheus-service
+  #       image: solidnerd/prometheus-service # no tag present!
```

Comment:
Thanks for adding helm here. I will take care of configuring skaffold.yaml properly for helm support, same as we have it in https://github.com/keptn/keptn/blob/master/helm-service/skaffold.yaml